File: string.py
print("Hello") #''와 ""는 차이가 없다
print('Hello') 
print("Hello 'World'") #"''"큰 따옴표 안에 작은 따옴표를 넣으면 작은 따옴표 출력 가능
print('Hello "World"') #'""'작은 따옴표 안에 큰 따옴표를 넣으면 큰 따옴표 출력 가능

print('Hello '+'world') #문자열 + 문자열
print('Hello '*3) # 3번 반복
print('Hello'[0]) # 문자열은 배열 처럼 들어가 있다.
print('Hello'[1])
print('Hello'[2])

print('hello world'.capitalize()) # capitalize() 첫 번째를 대문자로 변경
print('hello world'.upper()) # upper() 전체를 대문자로 변경
print('hello world'.__len__()) #__len__() 문자열 개수 세기
print(len('hello world')) #len('') 문자열 개수 세기
print('Hello world'.replace('world', 'programming')) #replace 앞의 문자열과 뒤의 문자열을 교체 한다.

print("egoing's \"tutorial\"") # 문자열에서 탈주 문자가 된다
print("\\") # \역슬래쉬 를 쓰고 싶을때 닫히지 않은 문자열
print("Hello\nworld")
print("Hello\t\tworld")
print("\a")
print('Hello\nworld')

print(10+5) # 산술 연산자
print("10"+"5") # 문자 + 문자

File: string/string.rb
puts('Hello') # ''와 ""는 차이가 없음
puts("Hello")
puts("Hello 'World'") # ""안에 ''를 넣어야 '' 출력 가능
puts('Hello "World"') # ''안에 ""를 넣어야 "" 출력 가능

puts('Hello '+'world') # string + string
puts('Hello '*3) # Hello Hello Hello
puts('Hello'[0]) # H
puts('Hello'[1]) # e
puts('Hello'[2]) # l

puts('hello world'.capitalize()) # 맨 앞에 대문자로 수정
puts('hello world'.upcase()) # upcase 전체를 대문자로 수정
puts('Hello world'.length()) # length() 문자열 개수 세기
puts('Hello world'.sub('world', 'programming')) #sub 'world'를 'programming'으로 수정

puts("egoing's \"tutorial\"") # 문자열에서 탈주 한다.
puts("\\") # 특수기호 표시할 시 ""
puts("Hello\nworld")# 줄바꿈
puts("Hello\t\tworld") # tab
puts("\a") # 빈 공간
puts('Hello\nworld') # ''에서는 줄바꿈이 안된다

puts(10+5) #사칙 연산 +
puts("10"+"5") # 문자 + 문자
